lib/pokerscore/hand.rb

```diff
@@ -11,10 +11,8 @@ def <=>(hand2)
     if straight_high_card != nil
       return straight_high_card <=> hand2.straight_high_card 
     end
-    values = []
-    hand2_values = []
-    (1..4).each { |index| values += sets[index].sort }
-    (1..4).each { |index| hand2_values += hand2.sets[index].sort }
+    values = 4.downto(1).map { |index| sets[index].sort }
+    hand2_values = 4.downto(1).map { |index| hand2.sets[index].sort }
```

Comment:
I know these lines existed before hand, but this could be a good opportunity to be a bit of a refactor.

From a brief glance, it's not immediately clear to me what is happening here. Lines of code like that are great, but it's sometimes nice to wrap them in a method, even if just for the sole purpose of giving the operation a nice name. Code is written once, but it is ready many more times after that, so we want to optimize for readability when we can.

Both operations are on a `Hand`, so my gut feeling would be to turn this operation into an instance method.

Reply:
Sounds good, thanks James